src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -88,6 +64,9 @@ const NavHeader = (props) => {
 
   const AnimatedBlurView = Animated.createAnimatedComponent(BlurView);
 
+  const TopicDefaultIcon =
+    followType === 'incognito' ? TopicAnonDefaultIcon : TopicSignedDefaultIcon;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `TopicDefaultIcon` is now determined based on the `followType`. This change seems fine as long as the `followType` prop is always provided and has a valid value.
```diff
+ const TopicDefaultIcon =
+   followType === 'incognito' ? TopicAnonDefaultIcon : TopicSignedDefaultIcon;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->